python/lsst/ts/fbs/utils/maintel/make_fieldsurvey_scheduler.py

```diff
@@ -113,6 +113,64 @@ def add_field_surveys(
                 )
             )
 
+    def add_field_altaz_surveys(
+        self,
+        tier: int,
+        observation_reason: str,
+        science_program: str,
+        survey_name: str | None = None,
+        basis_functions: typing.List[BaseBasisFunction] = [],
+        sequence: str = "ugrizy",
+        nvisits: dict | None = None,
+        exptimes: dict | None = None,
+        nexps: dict | None = None,
+        detailers: typing.List[BaseDetailer] = [],
+        **kwargs: typing.Any,
+    ) -> None:
+        """Add a list of field surveys to the scheduler configuration.
+
+        Parameters
+        ----------
+        tier: `int`
+            Tier index used to control prioritization of surveys.
+        json_block: `str`
+            JSON BLOCK used to perform the observations.
+        observation_reason: `str`
+            Purpose of the observation, e.g., "science"
+        science_program: `str`
+            Name of the science program.
+        basis_functions: `list` of `BaseBasisFunction`
+            Basis functions provided to each field survey.
```

Comment:
The rest of the parameters were not included in the docstring. Was it on purpose?